@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -1038,6 +1038,11 @@ export interface FillTechniqueParams extends BaseTechniqueParams, PolygonalTechn
      * Width of the lines. Currently limited to the [0, 1] range.
      */
     lineWidth?: DynamicProperty<number>;
+
+    /**
+     * Whether to clip the polygon at the Tile boundary.
+     */
```

Comment:
Please mention what is the default value for clipping

Reply:
I guess the default is undefined?

Reply:
what do you mean is `undefined`? you should write in the documentation something like
` Whether to clip the polygon at the Tile boundary. Default value is false` or true if it is true.

Reply:
I removed this from the style, so setting to resolved.